dockerfiles/puppeteer-context/mina_daemon_puppeteer.py

```diff
@@ -111,7 +111,7 @@ def stop_daemon():
   Path('daemon-active').unlink()
   mina_process = None
   log("daemon fully stopped" )
-  log_plain('{"puppeteer_script_event": true, "event_type": "node_offline", "message":"daemon is being stopped by puppeteer script and is going offline"}')
+  log_plain('{"puppeteer_script_event": true, "puppeteer_event_type": "node_offline", "message":"daemon is being stopped by puppeteer script and is going offline"}')
```

Comment:
Would be best if we had a `log_event` function that does this for you with some more safety (you pass in an event type and message, the function builds a python dict and serializes it to json). This will make it easier to add new events to the puppeteer in the future.